### PR TITLE
FST: enlarge testing to include basic analysis checks

### PR DESCRIPTION
**Pull request by sawenzel**

### CODE REVIEW

File: o2-full-system-test.sh [hunk 1]
```diff
@@ -24,11 +24,16 @@ $O2_ROOT/prodtests/full_system_test_ci_extra_tests.sh
 popd
 rm -Rf $BUILDDIR/full-system-test-sim
 
+# during build, the Python paths are not setup (in contrast to a full module load env)
+# python is needed by the O2DPG scheduler
+export PYTHONPATH=${PYTHONPATH}:${PYTHON_MODULES_ROOT}/share/python-modules/lib/python/site-packages
+
```
Comment (TimoWilken) [inline o2-full-system-test.sh:30]: Would it be better to use the following in the frontmatter instead? Then the path is set for this recipe and for anything that may end up depending on this recipe, I think.
```yaml
append_path:
  PYTHONPATH: "$PYTHON_MODULES_ROOT/share/python-modules/lib/python/site-packages"
```

**Review by davidrohr — DISMISSED** [review]
fine with me if it works.

File: o2-full-system-test.sh (hunk 1 above)
Comment (sawenzel) [reply o2-full-system-test.sh:30]: I tried to put:
```
build_requires:
  - Python
  - Python-modules
append_path:
  PYTHONPATH: "$PYTHON_MODULES_ROOT/share/python-modules/lib/python/site-packages"
```
but it is not working correctly.

Comment (ktf) [reply o2-full-system-test.sh:30]: append_path is for the dependent of the recipe, not for the build itself, so it needs to be there in the `Python-modules` recipe, not in this one.

Comment (ktf) [reply o2-full-system-test.sh:30]: The issue is the LHAPDF recipe, which resets the PYTHONPATH.

**Review by ktf — CHANGES_REQUESTED** [review]
We should fix LHAPDF instead.

**Comment by ktf** [comment]
https://github.com/alisw/alidist/pull/3625 should fix it.

File: o2-full-system-test.sh [hunk 2]
```diff
@@ -24,11 +24,16 @@ $O2_ROOT/prodtests/full_system_test_ci_extra_tests.sh
 popd
 rm -Rf $BUILDDIR/full-system-test-sim
 
+# during build, the Python paths are not setup (in contrast to a full module load env)
+# python is needed by the O2DPG scheduler
+export PYTHONPATH=${PYTHONPATH}:${PYTHON_MODULES_ROOT}/share/python-modules/lib/python/site-packages
```
Comment (ktf) [inline o2-full-system-test.sh:29]:
```suggestion
```

**Comment by ktf** [comment]
#3625 is now merged, so if you could remove the PYTHONPATH part, we can merge this.

**Comment by sawenzel** [comment]
recipe updated. Should be ok now.

**Review by ktf — APPROVED** [review]

**Pull request merged**